nni/retiarii/evaluator/pytorch/lightning.py

```diff
@@ -36,6 +37,9 @@ class LightningModule(pl.LightningModule):
     See https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html
     """
 
+    running_mode: Literal['multi', 'oneshot'] = 'multi'
```

Comment:
is it possible that a lightning class can be used both for multi and oneshot?

Reply:
When I used this flag, I meant to check before two actions:

1. Whether to save an onnx graph.
2. Whether to report intermediate / final results.

But on a second thought now, maybe evaluator could figure it out by itself whether its inner module is a one-shot supernet. So this might be not needed.